react-native-libraries.json

```diff
@@ -8642,5 +8642,33 @@
     "android": true,
     "web": true,
     "expo": true
+  },
+  {
+    "githubUrl": "https://github.com/BabylonJS/BabylonReactNative",
+    "npmPkg": "@babylonjs/react-native",
+    "examples": [
+      "https://github.com/BabylonJS/BabylonReactNative/tree/master/Apps",
+      "https://github.com/BabylonJS/BabylonReactNativeSample"
+    ],
+    "ios": true,
+    "android": true,
+    "windows": true
+  },
+  {
+    "githubUrl": "https://github.com/BabylonJS/BabylonReactNativeSample",
+    "ios": true,
+    "android": true,
+    "windows": true,
+    "template": true
+  },
+  {
+    "githubUrl": "https://github.com/runtothedoor/rotating-cube-demo-babylon-rxn",
+    "images": [
+      "https://raw.githubusercontent.com/runtothedoor/rotating-cube-demo-babylon-rxn/main/cube.gif"
+    ],
+    "ios": true,
+    "android": true,
+    "web": true,
+    "template": true
```

Comment:
By `template` we understand not the demo apps, but the project templates not tight to the one library, rather a complex app setup for multiple use case like, for example:
* https://github.com/react-native-community/react-native-template-typescript
* https://github.com/leotm/react-native-template-new-architecture

Reply:
Sorry for not being clear enough, those two entries should be a part of `@babylonjs/react-native` examples array, instead of being a sole entries in the directory. 🙂 

Saying that, we have on our list to consider if we would like to have a demo apps directory and if yes, how we would like to tackle this topic, since mixing libraries with demo apps might cause unnecessary noise for the users, and probably with time demo apps will outnumber the libraries anyway.

There are at least few factors which we need to account for, for example, unfortunately statistically demo apps come with a lot more risk of getting outdated/unmaintained/abandoned than the libraries, especially with lively community around them, which at the end can lower the quality of RND results in time and would require more maintenance work to keep the list in shape.

Reply:
> Sorry for not being clear enough, those two entries should be a part of `@babylonjs/react-native` examples array, instead of being a sole entries in the directory. 🙂
> 
> Saying that, we have on our list to consider if we would like to have a demo apps directory and if yes, how we would like to tackle this topic, since mixing libraries with demo apps might cause unnecessary noise for the users, and probably with time demo apps will outnumber the libraries anyway.
> 
> There are at least few factors which we need to account for, for example, unfortunately statistically demo apps come with a lot more risk of getting outdated/unmaintained/abandoned than the libraries, especially with lively community around them, which at the end can lower the quality of RND results in time and would require more maintenance work to keep the list in shape.

Sure, no problems! 

Editing...